deploy/scripts/src/mobile/testSteps/backend.ts

```diff
@@ -54,10 +57,16 @@ export function getBiometricTokenV2 (sessionId: string): void {
     const biometricTokenUrl = buildBackendUrl('/biometricToken/v2', {
       authSessionId: sessionId
     })
+
+    const startTime = Date.now()
     const res = http.get(biometricTokenUrl, {
       tags: { name: 'GET /biometricToken/v2' }
     })
+    const endTime = Date.now()
+
     isStatusCode200(res)
+      ? transactionDuration.add(endTime - startTime)
+      : fail('Response Validation Failed')
```

Comment:
@raluca-alecu - what does the fail() do here? Does it stop our test from proceeding to the next step if a request fails?

Reply:
Immediately throw an error, aborting the current script iteration.

Reply:
That's great (and we probably wanted it there in place before, otherwise we'd get a whole bunch of failures if an early step failed) thank you.

Reply:
Do we need an argument into the fail function? I can't work out it's value here, but might be missing something. I.e. can we simply go fail()

Reply:
1. The argument is optional. I followed the implementation of fail in other tests from this repo.
2. If we calculate the error request duration, this will be displayed on the final results, and our max request time and average may be affected 🤔 
Maybe @Tom-Dann can give his opinion 🙏

Reply:
It's not very helpful to include failing requests in the response time metrics, so this implementation is good (and also aligns to what we are doing in other test scripts). Our response time NFRs should be based on passing requests only.

An alternative in the future may be to track the duration of every request and then add a tag based on whether the assertions passed. Similar to how the default `http_req_duration` has a `expected_response` boolean tag on every metric. That would still give you the option to filter on successful requests, and also potentially some insight into durations for errors too (e.g. if it errors and the duration is 30,000ms it's easy to see it was probably a timeout)

For now though, tracking this custom metric on only passing requests suits our needs IMO

Reply:
Yep I agree on using success requests for metrics.